dashboard-visualiser-superset/docker-compose.postgres.yml

```diff
@@ -0,0 +1,31 @@
+version: "3.9"
+
+services:
+  postgres-metastore-1:
+    image: bitnami/postgresql-repmgr:14
+    environment:
+      POSTGRESQL_USERNAME: ${SUPERSET_POSTGRESQL_USERNAME}
+      POSTGRESQL_PASSWORD: ${SUPERSET_POSTGRESQL_PASSWORD}
+      POSTGRESQL_DATABASE: ${SUPERSET_POSTGRESQL_DATABASE}
+      POSTGRESQL_POSTGRES_PASSWORD: ${SUPERSET_POSTGRESQL_POSTGRES_PASSWORD}
+      REPMGR_NODE_NETWORK_NAME: postgres-metastore-1
+      REPMGR_NODE_NAME: postgres-metastore-1
+      REPMGR_RECONNECT_INTERVAL: 3
+      REPMGR_PASSWORD: ${SUPERSET_REPMGR_PASSWORD}
+      REPMGR_PRIMARY_HOST: ${SUPERSET_REPMGR_PRIMARY_HOST}
+      REPMGR_PARTNER_NODES: ${SUPERSET_REPMGR_PARTNER_NODES}
+    volumes:
+      - "superset-postgres-data:/bitnami/postgresql"
+    deploy:
+      replicas: 1
+      resources:
+        limits:
+          memory: ${SUPERSET_POSTGRES_MEMORY_LIMIT}
+    networks:
+      default:
```

Comment:
That is intentional yes, since no outside of superset should be able to access the superset db. 

Down the line we may want to add it to a shared backup network if we decided to schedule backups for it.